### PR TITLE
[EM-143] Show overview info for department projects

### PR DESCRIPTION
**Pull request by SandroDamilano**
## What does this PR do?
It adds an overview to the department metrics page showing the amount of projects per language and relevance within that department.

image (2)

Resolves [#143](https://rootstrap.atlassian.net/browse/EM-143)

### CODE REVIEW

File: app/services/builders/department_overview.rb [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    def initialize(department)
+      @department = department
+    end
+
+    def call
+      department_overview = language_names.each_with_object({}) do |language, overview|
+        overview[language] = language_overview(language)
+      end
+
+      department_overview.with_indifferent_access
```
Comment (horacio) [inline app/services/builders/department_overview.rb:14]: Just sharing another shorter option...

```suggestion
      department_overview = language_names.index_with { |language|
        language_overview(language)
      }.with_indifferent_access
```

File: app/services/builders/department_overview.rb [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+      {
+        per_relevance: relevances_overview,
+        totals: relevances_overview.values.sum
+      }
+    end
+
+    def relevances_overview(language)
+      project_relevances.each_with_object({}) do |relevance, overview|
+        overview[relevance] = projects_by_language_and_relevance[[language, relevance]] || 0
+      end
```
Comment (horacio) [inline app/services/builders/department_overview.rb:42]: Here I think we can do the same as above using `index_with`...

```suggestion
      project_relevances.index_with do |relevance|
        projects_by_language_and_relevance[[language, relevance]] || 0
      end
```

File: app/services/builders/department_overview.rb (hunk 1 above)
Comment (SandroDamilano) [reply app/services/builders/department_overview.rb:14]: True! I like it!

File: app/services/builders/department_overview.rb (hunk 2 above)
Comment (SandroDamilano) [reply app/services/builders/department_overview.rb:42]: Oh, I didn't know about that method! Thank you!

**Pull request merged**